src/main/scala/org/ergoplatform/api/ApplicationApiRoute.scala

```diff
@@ -0,0 +1,55 @@
+package org.ergoplatform.api
+
+import io.circe.syntax._
+import akka.actor.ActorRefFactory
+import akka.http.scaladsl.server.Route
+import io.circe.Json
+import org.ergoplatform.http.api.{ApiCodecs, ErgoBaseApiRoute}
+import org.ergoplatform.nodeView.wallet.scanning.{ExternalAppRequest, ExternalApplicationStorage}
+import org.ergoplatform.settings.ErgoSettings
+import scorex.core.api.http.ApiError.BadRequest
+import scorex.core.api.http.ApiResponse
+import scorex.core.settings.RESTApiSettings
+
+import scala.util.{Failure, Success}
+
+/**
+  * This class contains methods to register / deregister and list external applications.
+  * See EIP-0001 (https://github.com/ergoplatform/eips/blob/master/eip-0001.md)
+  */
+
+final case class ApplicationApiRoute (ergoSettings: ErgoSettings)
+                          (implicit val context: ActorRefFactory) extends ErgoBaseApiRoute with ApiCodecs {
```

Comment:
Please pay attention to formatting.

Reply:
fixed in 151d5b4f2e5eea5e6e5e07260f7fa97c56ace28e